adapters/celestia/Cargo.toml

```diff
@@ -9,7 +9,7 @@ publish = false
 
 [dependencies]
 borsh = { workspace = true, features = ["bytes"] }
-bech32 = { workspace = true }
+bech32 = { workspace = true, default-features = true }
```

Comment:
I honestly don't know, but what is the point of having `default-features = true` explicitly

Reply:
`bech32` is a requirement of `sov-modules-core`, and it must be `default-features = false`. This way, the bare minimum we declare on the root workspace is with such state, and if the library is declared elsewhere, it is expected to have the default features as these crates won't split std/no-std by feature